pyaspeller/speller.py

```diff
@@ -355,3 +350,51 @@ def api_options(self):
         if self._ignore_roman_numerals:
             options |= 2048
         return options
+
+
+class Word(YandexSpeller):
+    """
+    Class for quick spelling of single word.
+    """
+
+    def __init__(self, *args, **kwargs):
+        # http://stackoverflow.com/questions/2215923/avoid-specifying-all-arguments-in-a-subclass
+        if 'text' in kwargs:
+            text = kwargs.pop('text')
+        else:
+            text = args[0]
+        super(Word, self).__init__(*args[1:], **kwargs)
```

Comment:
I think that composition is more cleaner then inheritance here:
`self._speller = YandexSpeller()`
Or Word class can be renamed into WordSpeller. 
`Word('texx').spell('blah') looks strange`